frontend/src/pages/SearchPage/components/SearchResults/SearchResults.jsx

```diff
@@ -20,20 +21,17 @@ export const SearchResults = () => {
     return () => {
       window.removeEventListener("resize", updateDimension);
     };
-  }, []);
+  }, [dispatch]);
```

Comment:
Uważam że ten useEffect powinien się uruchamiac tylko raz na mount bo teraz się uruchamia za każdym razem jak ktoś zmieni szerokość okna na większe i mniejsze przez samo zmienianie szerokości okna jest wysyłane zapytanie do backend-u

ale to już na nowym branchu
**Dobra robota**